msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -178,6 +178,26 @@ For more information on the features available see: [Features in the Public Deve
      "contoso.com",
      "mysite.someplace.com",
      "othersite.someplace.com"
+  ],
+   "configurableProperties": {
+       "type": "array",
+       "description": "A list of tenant configured properties for an app.",
+       "maxItems": 9,
+       "items": {
+         "enum": [
```

Comment:
nit: JSON indentation

Reply:
Hi @vganiga - Can you please elaborate on the above comment? Did not understand what needs to be done here.
Thank you.

Reply:
the JSON indentation seems to be out of place. the closing brackets needs proper alignment